strawberry/experimental/pydantic/utils.py

```diff
@@ -36,3 +43,57 @@ def get_private_fields(cls: Type) -> List[dataclasses.Field]:
             private_fields.append(field)
 
     return private_fields
+
+
+class DataclassCreationFields(NamedTuple):
+    """Fields required for the fields parameter of make_dataclass"""
+
+    name: str
+    type_annotation: Type
+    field: dataclasses.Field
+
+    @property
+    def to_tuple(self) -> Tuple[str, Type, dataclasses.Field]:
```

Comment:
`to_tuple` sounds like a method :D 

maybe can rename it to `as_tuple` or just `tuple` or remove the property, but can be done in another PR :)

Reply:
removed the property!